docs/topics/collection-elements.md

```diff
@@ -71,7 +71,7 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
 
-## Retrieving by condition
+## Retrieving with selector
```

Comment:
Why do you think it's better?
"by condition" seems more familiar for the general audience.

Reply:
Because I put in this section `firstNotNullOf()`, which takes the mapping selector, not a condition.

We use the term "selector" quite often in the documentation, although I agree that "condition" is more familiar. I can make the separate section, but then `firstNotNullOf` would be the only item there. WDYT?

Reply:
I'd create a separate new section because most of this section remains about "by condition".

Reply:
+1 for a separate section.

Also note the section titles, the first one is "Retrieve ...", the second is "Retrieving ...".

Reply:
Made the separate section "Retrieve with selector" and unified the form of titles.